discovery.md

```diff
@@ -537,17 +563,118 @@ When encoded in JSON, the response format MUST adhere to the following:
-schema definitions for implementing this API using OpenAPI and gRPC as
-illustrative examples.
-
-### HTTP Binding
-
-When using JSON, the HTTP `Content-Type` value MUST be `application/json`.
-
-...
+### Management APIs
+
+All of the API endpoints specified in this section MUST be supported by
+compliant Discovery Endpoint implementations that support being managed.
+
+#### Asynchronous Processing
+
+For any of the following API endpoints, if the Discovery Endpoint chooses
+to process the incoming request asynchronous then the following rules apply:
+- A `202 Accepted` MUST be returned to the request. This indicates that the
+  request has been accepted but not processed yet.
+- The `202 Accepted` response MUST include a `Location` HTTP Header that
+  points to a "status endpoint", which can be used to determine the status
+  of the original request. The HTTP Response body MAY be empty.
+- The `202 Accepted` response MAY include a `Retry-After` HTTP Header
+  indicating the time, in seconds, that the sender SHOULD wait before
+  querying the status endpoint.
+
+A `GET` MAY be sent to the status endpoint to determine the status of the
+original request. The following responses are defined:
+
+- `200 OK` indicates that the original request was successfully processed.
+  The repsonse MUST include an HTTP `Location` Header pointing to the
+  location of the Service was that action upon. The response body
+  MUST include the Service that was being operated on, unless it
+  has been deleted, then the response body MUST be empty.
+- `202 Accepted` indicates that the original request is still being processed.
+  The response body MAY be empty.
+- `406 Not Acceptable` to indicate that the original request failed to be
+  processed correctly. The HTTP response Body SHOULD include additional
+  information as to why it failed.
+
+Other responses are allowed, but not defined by this specification, however
+they are related to the `GET` itself and not the original request.
+
+How long a Discovery Endpoint supports requests to the status endpoint
+is an implementation choice, however, it MUST be available immediately
+upon return of the `202 Accepted` from the original request.
+
+#### `POST /services`
+
+This MUST add a Service to the list of Services available at this Discovery
+Endpoint. The Body of the request message MUST contain a valid single Service
+Entry, with the exception that it MUST NOT include an `id` attribute.
+The `id` MUST be generated by the Discovery Endpoint.
+
+The follow responses are defined by this specification:
+- `201 Created` if the new Service was added. The response MUST include
+  an HTTP `Location` Header with a value that points to the new Service.
+  The HTTP Response Body MUST include the JSON represenation of the
+  newly created Service.
+- `404 Not Found` if there is no Service with the specified `id`.
+
+Other responses are allowed, but not defined by this specification.
+
+#### `PUT /services/{id}`
```

Comment:
I was going to update this PR to try to deal with "import" vs "update" but I decided to open an issue instead so we can deal with it on its own.  I'm leaning towards having the client add a query parameter to indicate which they're doing. But if someone has any other ideas please speak up.

https://github.com/cloudevents/spec/issues/701